modules/components/pages/outputs/redpanda_migrator.adoc

```diff
@@ -244,14 +246,32 @@ Replication factor for created topics. This is only used when `replication_facto
 
 *Default*: `3`
 
+
+
+=== `translate_schema_ids`
+
+Encodes Redpanda schema IDs ready for consumption by an Apache Kafka broker.
+
+*Type*: `bool`
+
+*Default*: `true`
+
+=== `schema_registry_output_resource`
+
+The label of the xref:components:outputs/schema_registry.adoc[schema_registry output] to user for fetching schema IDs.
```

Comment:
We don't currently mention anything about using the schema_registry output with this output, or have I misunderstood?